app/controllers/concerns/hyku_addons/registrations_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module HykuAddons
+  class RegistrationsController < Devise::RegistrationsController
```

Comment:
Is this meant to be a module that gets included into `Devise::RegistrationsController` or a class that replaces it?  If a class, then it should be moved out of the `concerns` directory.  I think this could work really well as a module if the new and create overrides were removed in favor of a `before_action` which does the redirect or raises an error which gets handled in ApplicationController (by performing the redirect).  Probably the latter of those two would be the cleanest.  I can point to some examples if you'd like.

Reply:
Please :)

Reply:
Here's an example of the handling of the raised error.  Looks like it could live in the module instead of ApplicationController.
https://github.com/samvera/hydra-head/blob/master/hydra-core/app/controllers/concerns/hydra/controller/controller_behavior.rb#L11-L32